### ISSUE TITLE
Which is the official version of the plugin?

### ISSUE THREAD

**Issue by sammuggPS** [EddyVerbruggen/cordova-plugin-googleplus#205]
On April 2, 2016, @bau720123 asked:

hi @sammuggPS 
could I ask，which one is the official version (by git) ? (I know the npm is a old version for now)
yours or @EddyVerbruggen ?

because of yours and @EddyVerbruggen almost have the same code，in addition to this file have a lot of different
https://github.com/EddyVerbruggen/cordova-plugin-googleplus/blob/master/src/android/GooglePlus.java
https://github.com/PointSource/cordova-plugin-googleplusupdate/blob/master/src/android/GooglePlus.java

so...which one should I use or test...tks

**Comment by sammuggPS** [comment]
Hey @bau720123, thanks for asking!

First things first, in the future, please **_open a separate ticket_** if you have a question not related to a thread you were looking at.

To answer your question:
This whole project was started by @EddyVerbruggen, so he'd be the one to say what is "official" or not.

From my perspective, what it comes down to is that if you pull from GitHub, you will have the absolute latest code (if not officially a version) for this plugin. That's really the only difference between pulling from here or NPM.

However, I recommend that you install the plugin from GitHub for several reasons:
1) Both Android and iOS are using Google Sign-In, so there's consistency.
2) Usage of the plugin has been more clearly defined and documented
3) You can help us find any bugs or potential issues before the update gets pushed to NPM!

In short, GitHub has the most up to date code, NPM does not. There is more benefit to your using the GitHub version right now than the NPM version.

Happy Coding,
~Sam
